render/filters.go

```diff
@@ -73,25 +73,24 @@ func ComposeFilterFuncs(fs ...FilterFunc) FilterFunc {
 // Filter removes nodes from a view based on a predicate.
 type Filter struct {
 	Renderer
-	FilterFunc   FilterFunc
-	filterPseudo bool
```

Comment:
My idea was to add an `And` combinator (like `Complement`) and combine it with  `IsNotPseudo`  in the APITopologyOptions, instead of embedding `IgnorePseudo` in the filter definitions.

Something like

```go
func And(f FilterFunc, g FilterFunc) FilterFunc {
	 return func(node report.Node) bool { return f(node) && g(node)}
}

option :=  APITopologyOption{"stopped", "Stopped containers", render.And(render.IsStopped, render.IsNotPseudo)}

```

Would that make the code simpler? I think that could make it cleaner but feel free to disregard it if you think it's too complex.